membership-attribute-service/conf/logback.xml

```diff
@@ -19,7 +19,7 @@
         </rollingPolicy>
 
         <encoder>
-            <pattern>%date %file:%L [%level]: %msg%n%xException{3}</pattern>
+            <pattern>%date %file:%L [%level]: %msg%n%xException{full}</pattern>
```

Comment:
we were only getting the first 3 lines of any exception in the logs, now we will get them all